pubspec.yaml

```diff
@@ -11,7 +11,7 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
```

Comment:
How does the SDK constraint work with this package and the analyzer dep? Would everything here technically work SDK v2.3.0 installed?

Reply:
I *think* so, yes.